gnocchi/tests/functional/gabbits/archive-rule.yaml

```diff
@@ -158,6 +158,32 @@ tests:
         authorization: "basic YWRtaW46"
       status: 400
 
+# rename an archive policy rule
+
+    - name: rename archive policy rule
+      PATCH: /v1/archive_policy_rule/test_rule3
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
+      data:
+          name: test_rule3_renamed
+      status: 200
+
+    - name: get renamed archive policy rule
+      GET: /v1/archive_policy_rule/test_rule3_renamed
+      status: 200
+
+# rename an archive policy rule with an already existing name
+
+    - name: rename archive policy rule with existing name
+      PATCH: /v1/archive_policy_rule/test_rule2
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
+      data:
+          name: test_rule3_renamed
+      status: 400
+
```

Comment:
could you maybe add a check to verify test_rule3 (original) doesn't exist after rename

Reply:
Done, added a test with missing name parameter too